optimum/exporters/tasks.py

```diff
@@ -637,9 +637,11 @@ def determine_framework(model: str, framework: str = None) -> str:
             4. If could not infer, use available framework in environment, with priority given to PyTorch.
 
         Args:
-            model (`str`):
+            model_name (`str`):
                 The name of the model to export.
-            framework (`str`, *optional*):
+            subfolder (`str`, *optional*, defaults to `""`):
+                The subfolder where the model is stored.
+            framework (`Optional[str]`, *optional*):
```

Comment:
disagree on `Optional` in docstring as above, imo should be

```
framework (`str`, *optional*, defaults to `None`)
```
or
```
framework (`str`, *optional*)
```

Reply:
`framework` is of type `Optional[str]`, it is a keyword argument, so "optional" in the sense of the docstring, and it defaults to `None` so we do not specify it.

Reply:
Average user does not know what typing's `Optional` means. I don't see the need to add this complexity in the doc.